spec/requests/api/v1/listings_spec.rb

```diff
@@ -130,24 +130,24 @@
     expect(json['ami'].length).to eq(9)
   end
 
-  it 'gets Lottery Preferences' do
-    VCR.use_cassette('listings/lottery-preferences') do
-      get '/api/v1/listings/lottery-preferences.json'
-    end
+  # it 'gets Lottery Preferences' do
```

Comment:
@kaptron this doesn't return anything anymore by salesforce and is blank on the listing page -- should I report this as a bug on Salesforce Pivotal or is this just stale code? I'm guessing the former